crates/nargo_cli/tests/test_data_ssa_refactor/brillig_not/src/main.nr

```diff
@@ -0,0 +1,11 @@
+// Tests a very simple program.
+// 
```

Comment:
"Tests a very simple program." could be more useful comment, maybe at least "Tests a very simple Brillig function"

Reply:
We can sweep through the other examples and change it since this is copied into every example and becomes quite redundant as we only care about what is being tested.